docs/source/ext/github_tools.py

```diff
@@ -50,7 +54,8 @@ def fetch_url(url):
         # url = Request(url,
         #               headers={'Accept': 'application/vnd.github.v3+json',
         #                        'User-agent': 'Defined'})
-        f = urlopen(req)
+        if url.lower().startswith('http'):
+            f = urlopen(req)
```

Comment:
What happens if it does not start with http? here a proposition:

```python
if not url.lower().startswith('http'):
    msg = "custom message"
    raise ValueError(msg)
f= urlopen(req)
```

The `ValueError` will need to be tested with `npt.assert_raises`

Reply:
Nice!
It didn't even cross my mind !!!
I will change in next commit, thank you for this.